### DISCUSSION TITLE
Change subscription after poll method invocation.

### DISCUSSION THREAD

**Discussion by AhmetBasar** [confluentinc/parallel-consumer#413] [Q&A] [answered]
Hello,

Is it possible to change subscription(i.e. start listening new topic) after poll method invocation.

Thanks.

**Comment by astubbs** [comment, marked as answer]
Hi, welcome to the project!

No - due to the consumer not being thread safe, not at the moment. However, it's a planned feature! Can be tracked here:
- #187

Related / due to:
- #186 

Workaround:

1. Close PC, 
2. then open it again, 
3. change subscription, 
4. poll.
